src/status_im/ui/screens/intro/views.cljs

```diff
@@ -217,108 +244,357 @@
       (i18n/label :t/processing)]]
     [password-container confirm-failure? view-width]))
```

Comment:
```suggestion
    [password-container confirm-failure? view-width (not processing?)]))
```

Reply:
well it doesn't work like this because processing? is false again but maybe there's a solution with an atom and the confirmation button? Anyway the notif screen will go away so doesn't matter. We should merge

Reply:
FTR I got it to work by rewriting it like this:

```
(defn confirm-code []
  (let [auto-focus? (atom true)]
    (fn [{:keys [confirm-failure? processing? view-width]}]
      (if processing?
        (do (reset! auto-focus? false)
            [react/view {:style {:justify-content :center
                                 :align-items :center}}
             [react/activity-indicator {:size      :large
                                        :animating true}]
             [react/text {:style {:color      colors/gray
                                  :margin-top 8}}
              (i18n/label :t/processing)]])
        [password-container confirm-failure? view-width @auto-focus?]))))
```

Reply:
I think the problem is due to the fact that when processing? is done password-container is re-rendered

Reply:
maybe a cond in intro-step-forward for when account is created that to a navigation reset? because currently the problem is that you can navigate back and you shouldn't be able to

Reply:
That's a great hint, tried it and it works, and also fixes the keyboard issue. Thanks @yenda :)